apis/task.py

```diff
@@ -48,13 +48,20 @@ class TpuTask(BaseTask):
     task_test_config: Test configs to run on this TPU.
     task_gcp_config: Runtime TPU creation parameters.
     task_metric_config: Metric configs to process metrics.
+    custom_tpu_name: A custom TPU name.
```

Comment:
Can you add a note that the default is the task name?

Reply:
Sure thing.